libraries/AP_HAL_Linux/RCOutput_AioPRU.cpp

```diff
@@ -53,8 +61,16 @@ void RCOutput_AioPRU::init()
    // Reset PRU
    *ctrl = 0;
 
-   // Load firmware
+   // Loading firmware
+   // for ARM64 using loading PRU code in block instead memcpy
+#ifdef PRU_LOAD_NOT_USE_MEMCPY
```

Comment:
```suggestion
#if AP_HAL_LINUX_AIOPRU_LOAD_WORDWISE
```
(or invert the code and `AP_HAL_LINUX_AIOPRU_LOAD_USING_MEMCPY`)

It's almost always a mistake to create defines with the word "NOT" or "NO" in them.